src/index.html

```diff
@@ -4,13 +4,48 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, initial-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Search bar airbnb</title>
+    <title>Чорновик</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form 
+      action="*" 
+      method="post"
+      data-qa="big"
+      class="form"
+      >
+        
+      <input 
+        type="search" 
+        placeholder="Try &quot;Los Angeles&quot; " 
+        data-qa="keypress"
+        class="input"
+      >
+    </form>
+
+    <form 
+      action="*" 
+      method="post"
+      data-qa="small"
+    >
+      <input 
+        type="search" 
+        placeholder="Try &quot;Los Angeles&quot;"
+        class="input input--small"
+      >   
+    </form>
+
+    <img 
+      src="images/Search.svg" alt="search" 
+      class="photo_second">
+
+    <img 
+      src="images/Search.svg" alt="search2"
+      class="photo_first">
```

Comment:
```suggestion
    <img 
      src="images/Search.svg" alt="search" 
      class="photo_second"
    >

    <img 
      src="images/Search.svg" alt="search2"
      class="photo_first"
    >
```
Move this images inside `form` apply `position: relative` to the form and `position: absoulte` to the images